src/EnergyPlus/DXCoils.cc

```diff
@@ -7500,8 +7492,8 @@ namespace DXCoils {
 				OutdoorWetBulb = OutWetBulbTemp;
 			}
 			if ( DXCoil( DXCoilNum ).IsSecondaryDXCoilInZone ) {
-				OutdoorDryBulb = Node( DXCoil( DXCoilNum ).SecZoneAirNodeNum ).Temp;
-				OutdoorHumRat = Node( DXCoil( DXCoilNum ).SecZoneAirNodeNum ).HumRat;
+				OutdoorDryBulb = ZT( DXCoil( DXCoilNum ).SecZonePtr );
+				OutdoorHumRat = ZoneAirHumRat( DXCoil( DXCoilNum ).SecZonePtr );
 				OutdoorWetBulb = DXCoil( DXCoilNum ).EvapInletWetBulb;
 				OutdoorPressure = OutBaroPress;
 			}
```

Comment:
Sorry - not following this comment.  But I would say for now, keep the fix as minimal as possible.